src/features/profile/services/profile.service.ts

```diff
@@ -0,0 +1,52 @@
+import { BACKEND_V2_URL } from "app/config";
+import { BackendError } from "app/exceptions";
+import axios from "axios";
+import { User } from "core/users/types";
+
+const URL = `${BACKEND_V2_URL}/profiles`;
+
+export async function fetchOneProfile(
+  token: string,
+  profile_id: number
+): Promise<OneProfileResponse> {
+  try {
+    const response = await axios.get(URL + "/" + profile_id, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    return response.data;
+  } catch (error) {
+    throw new BackendError(error);
+  }
+}
+
+export async function updateProfile(
+  token: string,
+  body: updateProfileBody,
+  profile_id: number
+) {
```

Comment:
Mas de 3 argumentos en una funcion 😨? 

El tio Bob sataniza eso, recomienda usar un objeto a partir de 2 parametros
```suggestion
export async function updateProfile({
  token,
  body,
  profileId
}: {
  token: string,
  body: updateProfileBody,
  profileId: number
}) {
```

Reply:
para ese momento el update no estaba listo, por lo que eso lo tomare en cuenta para el PR que subire despues

Reply:
en esta branch no solucione el update de perfil, solo el mostrar.